zproject/settings.py

```diff
@@ -965,6 +965,14 @@ def get_secret(key):
     POPULATE_PROFILE_VIA_LDAP = 'zproject.backends.ZulipLDAPAuthBackend' in AUTHENTICATION_BACKENDS or POPULATE_PROFILE_VIA_LDAP
 
 ########################################################################
+# GITHUB AUTHENTICATION SETTINGS
+########################################################################
+SOCIAL_AUTH_USER_MODEL = AUTH_USER_MODEL
+SOCIAL_AUTH_GITHUB_KEY = ''
+SOCIAL_AUTH_GITHUB_SECRET = ''
+SOCIAL_AUTH_GITHUB_SCOPE = ['email']
```

Comment:
All but the first one of these should probably be handled consistent with http://zulip.readthedocs.io/en/latest/settings.html (as either a user-configurable value in prod_settings_template.py or a secret)